packages/twenty-front/src/modules/settings/developers/components/SettingsApiKeysFieldItemTableRow.tsx

```diff
@@ -39,7 +39,7 @@ export const SettingsApiKeysFieldItemTableRow = ({
       <TableCell
         color={
           fieldItem.expiration === 'Expired'
-            ? theme.font.color.danger
```

Comment:
this should not be necessary if we keep font.color.danger = color.red, see comment below